src/app/globals.css

```
@@ -2,6 +2,7 @@
 @tailwind components;
 @tailwind utilities;
 
+/* --- Base Styles & ShadCN Theme --- */
```

multiline comment on single line

```suggestion
// Base Styles & ShadCN Theme
```